File: tests/human/x/ts/order_by_map.ts
const data = [
  { a: 1, b: 2 },
  { a: 1, b: 1 },
  { a: 0, b: 5 },
];
const sorted = data
  .slice()
  .sort((x, y) => {
    if (x.a !== y.a) return x.a - y.a;
    return x.b - y.b;
  });
console.log(sorted);

File: tests/human/x/dart/order_by_map.dart
void main() {
  var data = [
    {'a': 1, 'b': 2},
    {'a': 1, 'b': 1},
    {'a': 0, 'b': 5}
  ];
  var sorted = List.from(data)
    ..sort((x, y) {
      var cmpA = (x['a'] as int).compareTo(y['a'] as int);
      if (cmpA != 0) return cmpA;
      return (x['b'] as int).compareTo(y['b'] as int);
    });
  print(sorted);
}
